cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/generator/CobiGenWrapper.java

```diff
@@ -564,10 +560,6 @@ private boolean isMergableFile(Set<TemplateTo> templates) {
      */
     public boolean isOverridableFile(String path, Collection<IncrementTo> consideredIncrements) {
         if (path != null) {
-            if (consideredIncrements.contains(new ComparableIncrement(ALL_INCREMENT_ID, ALL_INCREMENT_NAME, null,
```

Comment:
I had a call with Mandy to check this, and it seems that in a special case it throws a warning saying that increments got duplicated (it still generates). So she will investigate this further.